department-of-veterans-affairs/abd-vro
# Removing refs to self-hosted Postgres

Followup PR to https://github.com/department-of-veterans-affairs/abd-vro/pull/2040
 


## What was the problem?

There were many references in our deployment infrastructure to a self-hosted Postgres instance which we will be moving away from in favor of RDS going forward.

Associated tickets or Slack threads:
- #1845 

## How does this fix it?[^1]

Removes the references to the self-hosted Postgres in our deployment processes to simplify understanding and reduce code complexity.

## How to test this PR
- Only removes functionality


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
